### ISSUE TITLE
Click on 'OK' in production does not do anything

### ISSUE THREAD

**Issue by SophiaPetrova** [infinum/cookies_eu#73]
Hey! Click on 'OK' button works as expected locally but cannot be clicked on production in Chrome and Firefox. I am using Rails 5.1.0 without turbolinks

**Comment by gabskoro** [comment]
That's weird. Do you get any error in the console? I assume the assets are precompiled as they should be...

**Comment by SophiaPetrova** [comment]
Yes, they are. I found a workaround by putting `cookiesEu.init()` in the end of application.js

**Comment by rafapoza** [comment]
Have you tried to put the requires in the order you really require them?

![image](https://user-images.githubusercontent.com/4456103/98550654-aa292d80-229c-11eb-88b2-91ea44cac9d3.png)

This is working for me
